test/scripts/dump-users.js

```diff
@@ -0,0 +1,190 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const ROOT_DIR = '../..'
+
+const cp = require('child_process')
+const { assert } = require('chai')
+const path = require('path')
+const P = require('bluebird')
+const mocks = require(`${ROOT_DIR}/test/mocks`)
+
+const cwd = path.resolve(__dirname, ROOT_DIR)
+cp.execAsync = P.promisify(cp.exec)
+
+const log = mocks.mockLog()
+const config = require('../../config').getProperties()
+const Token = require('../../lib/tokens')(log, config)
+const UnblockCode = require('../../lib/crypto/random').base32(config.signinUnblock.codeLength)
+
+describe('scripts/dump-users', () => {
+  let db;
+
+  before(() => {
+    const DB = require('../../lib/db')(
+      config,
+      log,
+      Token,
+      UnblockCode
+    )
+
+    return DB.connect(config[config.db.backend])
+      .then(_db => {
+        db = _db;
+        return db.createAccount({
```

Comment:
These tests fail on travis because it cannot connect to the DB. Work aroundable?